tests/multicomponent_compressible_averaging_phases.prm

```diff
@@ -0,0 +1,42 @@
+# This test checks that the addition of phase transitions with corresponding
+# change material properties works correctly in the multicomponent material
+# model. The test is a modification of tests/multicomponent_compressible_averaging.prm,
+# with the primary changes including the naming of a compositional field, a 
+# phase transition, and corresponding changes in material properties.
+include $ASPECT_SOURCE_DIR/tests/multicomponent_compressible_averaging.prm
```

Comment:
The multicomponent_compressible_averaging test has a very nice description of what output to expect (i.e., what's the expected result of the averaging). I think you changed the parameters in the lower half of the box, so would it make sense to write a short description here about what material numbers to expect in the output, just so that we know the test results are correct?